### ISSUE TITLE
How to grab icon properties of another file type

### ISSUE THREAD

**Issue by cbcoutinho** [file-icons/atom#545]
Hello there,

I would like to match the icon of a certain undefined filetype to one that is defined elsewhere. I'm guessing there are a couple ways of doing this:

1. Assign the file type extension to a grammar in `config.cson`, given you know the target file source type (I don't know how to determine sources though)
2. Put a custom icon style in `styles.less` associated with the extension. I can try to imitate another icon type by hardcoding the rules (e.g. font-size, content, color), but I would rather just say: use the icon settings from this icon source.

In short: How do I set the file-icons of a specific file extension to match another already known file type/source?

What's the best way to accomplish what I'm trying to do?

**Comment by Alhadis** [comment]
> (I don't know how to determine sources though)

What do you mean? Are you referring to grammar scopes? (e.g., `source.python`, `source.js`, `text.html.basic`). If so, you can find the scope-name of a language by opening its package's settings in Atom:

<img src="https://cloud.githubusercontent.com/assets/2346707/23030834/115e8042-f4c3-11e6-9cd4-5a39ef0bb5a0.png" width="274" />

So to declare `.foo` as a JavaScript extension, you'd add this to your user config:

~~~coffee
core:
	customFileTypes:
		"source.js": [
			"foo"
		]
~~~

Failing that, you can always hardcode the styles in `styles.less`.

What's the extension/language you're trying to add?

**Comment by cbcoutinho** [comment]
Yes I'm referring to grammar scopes, but I don't have the language installed for the files I'm talking about - namely compiled c and Fortran files.

For c you mostly have `.o` object files, which Fortran compilers also produce. I don't have a `language-object` installed anywhere, so I'm not sure why those are now receiving a red text icon with 1s and 0s.

Further, depending on how you organize your program, a Fortran compiler can also produce `.mod` and `.smod` files, which are essentially compiled header files. Currently the `.mod` files are being given an icon called `modula2' [config.cson L2424-L2440](https://github.com/file-icons/atom/blob/master/config.cson#L2434-L2440). I have no clue what that is, but I have gotten used to those icons and I would like to apply the same style to the `.smod` files as well.

I guessed, and from your advice I added the following lines to my `config.cson`:
```
"*":
  core:
      "source.modula-2": [
        "mod"
        "smod"
      ]
```

Thanks for the help
